### PR TITLE
Update version numbers

### PR DESCRIPTION
**Pull request by jskeet**
- GAX version updated to 1.0.0-beta06 everywhere
- Package version numbers updated for next release, including
  promoting Trace, Monitoring, ErrorReporting and AspNet to beta

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,16 +19,16 @@ The following libraries are available at a [beta](#versioning) quality level:
 
 * [Google Cloud Pub/Sub](https://cloud.google.com/pubsub/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.PubSub.V1/) (beta)
 * [Google Cloud Natural Language](https://cloud.google.com/natural-language/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Language.V1/) (beta)
+* [Stackdriver Monitoring](https://cloud.google.com/monitoring/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Monitoring.V3/) (alpha)
+* [Stackdriver Error Reporting](https://cloud.google.com/error-reporting/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.ErrorReporting.V1Beta1/) (alpha)
+* [Stackdriver Trace](https://cloud.google.com/trace/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Trace.V1/) (alpha)
+* [Google Cloud Diagnostics for ASP.NET] - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Diagnostics.AspNet/) (alpha)
```
Comment (chrisdunelm) [inline README.md]: beta?

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
   },
 
   "dependencies": {
-    "Google.Api.Gax": "1.0.0-beta05",
+    "Google.Api.Gax": "1.0.0-beta06",
```
Comment (chrisdunelm) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: Does the version number of this library need to change from alpha01 to alpha02/beta01?

File: README.md [hunk 3]
```diff
@@ -19,16 +19,16 @@ The following libraries are available at a [beta](#versioning) quality level:
 
 * [Google Cloud Pub/Sub](https://cloud.google.com/pubsub/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.PubSub.V1/) (beta)
 * [Google Cloud Natural Language](https://cloud.google.com/natural-language/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Language.V1/) (beta)
+* [Stackdriver Monitoring](https://cloud.google.com/monitoring/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Monitoring.V3/) (alpha)
```
Comment (chrisdunelm) [inline README.md]: beta?

File: README.md [hunk 4]
```diff
@@ -19,16 +19,16 @@ The following libraries are available at a [beta](#versioning) quality level:
...
+* [Stackdriver Error Reporting](https://cloud.google.com/error-reporting/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.ErrorReporting.V1Beta1/) (alpha)
```
Comment (chrisdunelm) [inline README.md]: beta?

File: README.md [hunk 5]
```diff
@@ -19,16 +19,16 @@ The following libraries are available at a [beta](#versioning) quality level:
...
+* [Stackdriver Trace](https://cloud.google.com/trace/) - [API docs](http://googlecloudplatform.github.io/google-cloud-dotnet/docs/Google.Cloud.Trace.V1/) (alpha)
```
Comment (chrisdunelm) [inline README.md]: beta?

File: README.md (hunk 3 above)
Comment (jskeet) [inline README.md]: Doh, thanks, fixed.

File: apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json (hunk 2 above)
Comment (jskeet) [inline apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/project.json]: No, it hasn't been released yet.

**Pull request merged**